app/controllers/user_targets_controller.rb

```diff
@@ -16,6 +16,15 @@ def create
     end
   end
 
+  def show
+    @target = UserTarget.find(params[:id]);
+  end
+
+  def destroy
+    @target = UserTarget.find(params[:id]);
```

Comment:
You can use a helper method to load the `UserTarget` since this code is repeated twice and I assume is used in the view. 
https://apidock.com/rails/ActionController/Helpers/ClassMethods/helper_method